packages/palette/src/elements/index.tsx

```diff
@@ -16,7 +16,7 @@ export * from "./Input"
 export * from "./Join"
 export * from "./Menu"
 export * from "./Message"
-export * from "./Modal"
+export * from "./Modal/Modal"
```

Comment:
Totally understand the index-files-are-weird argument but we do have a convention in palette and it's used throughout. If we want to change that convention we should do so in a separate PR that a) argues reasoning; and b) updates everything, rather than slipping in one-off changes like the above.

Reply:
Yeah naming/structural consistency is more important IMO :+1:

We could just delete all the index files and get the best of both worlds at the cost of one weird diff, but I don't feel like it matters much either way. Happy for this one index file to be reinstated as long as it's just a single export statement like the others.